run.py

```diff
@@ -316,5 +316,9 @@ def process(
         logger.info('Done running 👆', extra=extra)
 
 
-if __name__ == '__main__':
+def main():
     fire.Fire(process)
```

Comment:
run.py is still useful?

Reply:
Yep, it was one of my goals here to get everything working with as little changes as possible!

But see https://github.com/mne-tools/mne-bids-pipeline/pull/611#issuecomment-1265968499